lib/constants/constants.dart

```diff
@@ -1 +1,9 @@
+import 'package:flutter/material.dart';
+
 const String localhost = '10.0.2.2:8080';
+const Map<String, String> headers = {
+  'Content-Type': 'application/json; charset=UTF-8',
+};
+Color bgColor = const Color.fromARGB(255, 48, 48, 48);
+TextStyle tilesText = const TextStyle(color: Colors.white, letterSpacing: 2.0);
+Color iconColor = const Color.fromARGB(255, 173, 173, 173);
```

Comment:
Nice. Keep expanding on this where appropriate. Your `Widget`s will read like a book if they use the same common values that are all named in a descriptive way.